doc/cluster_spec.md

```diff
@@ -24,7 +24,9 @@ Some options in a running cluster specification can be changed to update the des
 | maxSynchronousStandbys    | maximum number of required synchronous standbys when synchronous replication is enabled (only set this to a value > 1 when using PostgreSQL >= 9.6)                                                                                                                                                                                                                                                                                                                               | no                        | uint16            | 1                                                                                                                                   |
+| additionalSyncStandbyNames| additional names for possible standbys not included into stolon cluster, but used as replication destinations                                                                                                                                                                                                                                                                                                                                                                     | mo                        | string            | null                                                                                                                                |                    
 | additionalWalSenders      | number of additional wal_senders in addition to the ones internally defined by stolon, useful to provide enough wal senders for external standbys (changing this value requires an instance restart)                                                                                                                                                                                                                                                                              | no                        | uint16            | 5                                                                                                                                   |
+| additionalReplicationSlotName | name of the additional replication slot which is always created by the master postgres instance                                                                                                                                                                                                                                                                                                                                                                               | no                        | string            | null                                                                                                                                |
```

Comment:
Cannot be this a `[]string` as above so users can define multiple additional replication slots?